packages/salesforcedx-vscode-core/test/vscode-integration/commands/baseDeployRetrieve.test.ts

```diff
@@ -26,8 +27,9 @@ import {
 } from '@salesforce/source-deploy-retrieve/lib/src/client/types';
```

Comment:
Can these be referenced from the top level of `@salesforce/source-deploy-retrieve`?

Reply:
Great point! I have updated `ComponentStatus` and `MetadataApiRetrieveStatus` so that they are referenced from the top level. I'll update `MetadataApiDeployStatus` once the SDR version is bumped.